guides/hosting/performance/caches.md

```diff
@@ -15,83 +15,60 @@ The HTTP Cache is a *must-have* for every production system. With an enabled cac
 
 ### How to configure the HTTP cache
 
-The HTTP cache configuration takes place completely in the `.env file.` The following configurations are available here:
+The HTTP cache configuration takes place completely in the `.env.local` file. The following configurations are available here:
 
 | Name                          | Description                    |
 |:------------------------------|:-------------------------------|
 | `SHOPWARE_HTTP_CACHE_ENABLED` | Enables the HTTP cache         |
 | `SHOPWARE_HTTP_DEFAULT_TTL`   | Defines the default cache time |
 
-### How to change the cache storage
+The storage used for HTTP Cache is always the [App Cache](#app-cache), see below how to configure it. If you want to move this out of the application cache, you should use an external reverse proxy cache like [Varnish](https://varnish-cache.org/) or [Fastly](https://www.fastly.com/). For more [see here](../infrastructure//reverse-http-cache.md).
 
-The standard Shopware HTTP cache can be exchanged or reconfigured in several ways. The standard cache comes with an `adapter.filesystem`. The configuration can be found in the `shopware/src/Core/Framework/Resources/config/packages/framework.yaml` file.
+## How to change the cache storage
 
-```yaml
-framework:
-    cache:
-        pools:
-            cache.http:
-                adapter: cache.adapter.filesystem
-```
+The standard Shopware HTTP cache can be exchanged or reconfigured in several ways. The standard cache comes with an `adapter.filesystem`. This is a file-based cache that stores the cache in the `var/cache` directory. This allows Shopware to work out of the box on a single server without any additional configuration. However, this is maybe not the best solution for a production system, especially if you are using a load balancer or multiple servers. In this case, you should use a shared cache like [Redis](https://redis.io/).
```

Comment:
```suggestion
The standard Shopware HTTP cache can be exchanged or reconfigured in several ways. The standard cache comes with an `adapter.filesystem`. This is a file-based cache that stores the cache in the `var/cache` directory. This allows Shopware to work out of the box on a single server without any additional configuration. However, this may not be the best solution for a production system, especially if you are using a load balancer or multiple servers. In this case, you should use a shared cache like [Redis](https://redis.io/).
```